app/javascript/components/intro/components/list/constants.js

```diff
@@ -0,0 +1,16 @@
+const videos = {
```

Comment:
For me, it would be better an array... but you can also work with an object. Are those ids titles and titles ids?